### ISSUE TITLE
Should simple-cipher come before crypto-square?

### ISSUE THREAD

**Issue by ldwoolley** [exercism/xjavascript#312]
Today, I was working the simple-cipher and I noticed that the description mentions that we will learn an improvement to simple-cipher later on when we tackle crypto-square.  Currently, crypto-square comes three slots before simple-cipher.  Seems to me that we should switch the order of the exercises or change the description in simple-cipher.  Maybe it can simply mention that crypto-square improves the algorithms strength against frequency analysis.  Yea this is a simple, and trivial item.  And I will submit a proposed fix, once there is some agreement on a path.

I recommend that we propose a change to the description in x-commons, so that the exercises can be worked in order of challenge, similar to the goals of issue #299.  I would change it to a reference to crypto-square, and let each language track pick it's own order.

**Comment by rchavarria** [comment]
Yeah. I like those ideas (let me rephrase them just to be sure I understand you):

1. Modify simple-cipher description in x-common to stop referencing crypto-square, or create an issue to discuss it. I like the idea of having independent exercises, so I'd buy this one.
2. Reorganize simple-cipher and crypto-square exercises in this track (and probably on xecmascript), while letting other tracks to decide. In the meantime, as you have recently worked on these problems, you could fill in the `topics` property in `config.json` on each exercise, to have some idea about what topics are related to these exercises (this would be closely related to #299).

**Comment by ldwoolley** [comment]
Sorry, I lost track of this one.  But like a lost coin, it will be nice to do something with it.  Yes these were what I thought were the two options.  Which one did you want to do? I slightly prefer the option to make them stand alone.

**Comment by rchavarria** [comment]
You can propose a PR in [x-common](), to stop mentioning crypto-square from simple-cipher. Then, you can place simple-cipher before crypto-square. And...

> In the meantime, as you have recently worked on these problems, you could fill in the topics property in config.json on each exercise, to have some idea about what topics are related to these exercises (this would be closely related to #299)

**Comment by ldwoolley** [comment]
Soon to be done.
